end-to-end-tests/cypress.config.ts

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const { defineConfig } = require("cypress");
```

Comment:
I've added this eslint ignore line here - although this may be better to convert into an import.